### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory

### ISSUE THREAD

**Issue by evandrocoan** [Monnoroch/ColorHighlighter#388]
After I updated my color scheme, color highlighter does not updated its copy of it. Them I deleted the color highlighter folder on the User folder to force it to reload/recreate its copy of my color scheme.

But after opening a file I got this on my console:
```
Traceback (most recent call last):
  File "D:\SublimeText\sublime_plugin.py", line 389, in run_callback
    expr()
  File "D:\SublimeText\sublime_plugin.py", line 428, in <lambda>
    run_callback('on_load', callback, lambda: callback.on_load(v))
  File "D:\SublimeText\Data\Packages\Color Highlighter\main.py", line 551, in on_load
    ColorHighlighterPlugin.components.provide_color_selection_event_listener().on_load(view)
  File "D:\SublimeText\Data\Packages\Color Highlighter\main.py", line 458, in on_load
    if not self._init_view(view):
  File "D:\SublimeText\Data\Packages\Color Highlighter\main.py", line 507, in _init_view
    view, color_scheme, ColorHighlighterPlugin.components.provide_fake_color_scheme())
  File "D:\SublimeText\Data\Packages\Color Highlighter\main.py", line 85, in set_fake_color_scheme
    copy_resource(color_scheme, fake_color_scheme_path)
  File "D:\SublimeText\Data\Packages\Color Highlighter\main.py", line 65, in copy_resource
    with open(destination_path, "wb") as file:
FileNotFoundError: [Errno 2] No such file or directory: 'D:\\SublimeText\\Data\\Packages\\User\\Color Highlighter\\themes\\Default Stylers.tmTheme'
```
I use Sublime Text build 3147, Color Highlighter should recreate its copy of the color scheme, if it is folder is deleted on the User folder.

**Comment by Monnoroch** [comment]
To make it more clear: the problem is that if the `themes` folder is deleted, writing the theme fails. The folder will be recreated on ST restart automatically, but it would be more convenient to recreate it every time.

**Comment by Monnoroch** [comment]
@evandrocoan fixed the problem in `8.0.9`.

**Comment by evandrocoan** [comment]
Thanks!
